configs/biggan/README.md

```diff
@@ -0,0 +1,82 @@
+# LARGE SCALE GAN TRAINING FOR HIGH FIDELITY NATURAL IMAGE SYNTHESIS
+
+## Introduction
+<!-- [ALGORITHM] -->
+```latex
+@inproceedings{
+brock2018large,
+title={Large Scale {GAN} Training for High Fidelity Natural Image Synthesis},
+author={Andrew Brock and Jeff Donahue and Karen Simonyan},
+booktitle={International Conference on Learning Representations},
+year={2019},
+url={https://openreview.net/forum?id=B1xsqj09Fm},
+}
+```
+
+The BigGAN/BigGAN-Deep is a conditional generation model that can generate both high-resolution and high-quality images by scaling up the batch size and number of model parameters.
+
+We have conducted training BigGAN with CIFAR10(3x32x32) and ImageNet1k(3x128x128) dataset, and the sampling results are showed below.
+<div align="center">
+  <b> Results from our BigGAN trained in CIFAR10</b>
+  <br/>
+  <img src="https://user-images.githubusercontent.com/22982797/126476913-3ce8e2c8-f189-4caa-90ed-b44e279cb669.png" width="800"/>
+</div>
+
+<div align="center">
+  <b> Results from our BigGAN trained in ImageNet</b>
+  <br/>
+  <img src="https://user-images.githubusercontent.com/22982797/127615534-6278ce1b-5cff-4189-83c6-9ecc8de08dfc.png" width="800"/>
+</div>
+
+Evaluation of our trained BIgGAN.
+|    Models    | Dataset |   FID (Iter) | IS (Iter) | Config |  Download  |
+|:------------:|:-------:|:--------------:|:---------------:|:------:|:----------:|
+| BigGAN 32x32 | CIFAR10 |     9.78(390000)           |       8.70(390000)          | [config](https://github.com/open-mmlab/mmgeneration/blob/master/configs/biggan/biggan_cifar10_32x32_b25x2_500k.py) | [model](https://download.openmmlab.com/mmgen/biggan/biggan_cifar10_32x32_b25x2_500k_20210728_110906-08b61a44.pth)\|[log](https://download.openmmlab.com/mmgen/biggan/biggan_cifar10_32_b25x2_500k_20210706_171051.log.json) |
+| BigGAN 128x128 | ImageNet1k |       12.32(1150000)         |         72.7(1150000)        | [config](https://github.com/open-mmlab/mmgeneration/blob/master/configs/biggan/biggan_imagenet1k_128x128_b32x8_1500k.py) | [model](https://download.openmmlab.com/mmgen/biggan/biggan_imagenet1k_128x128_b32x8_1150k_20210730_124753-b14026b7.pth)\|[log](https://download.openmmlab.com/mmgen/biggan/biggan_imagenet1k_128x128_b32x8_1500k_20210726_224316.log.json) |
+
+Note: This is an earlier version(1150k iter) of BigGAN trained on `ImageNet1k`, the model with best performance is still on the way.
```

Comment:
**Note:** This is an unfinished version (1150k iter) of BigGAN trained on `ImageNet1k`. The model with the best performance is still on the way.